### ISSUE TITLE
npm package contains package.appxmanifest 

### ISSUE THREAD

**Issue by gaspardip** [KenRmk/winrt-net#1]
VS complains when trying to deploy a solution with 2 package.appxmanifest.

A workaround is to include only the `lib` folder in the project, but still, I feel the npm package shouldn't include the mentioned file.

**Comment by MikeKovarik** [comment]
Yes, you're right. I should've included .npmignore file. Though I'm about to deprecate this module.

1) `winrt` name is out of the game now that we have uwp
2) I lost any hope in JSPM and SystemJS, in the years of trying to use it and build module for it, it brought me more problems and pain than good.
3) I'm expanding and refining this code to be used for serial and bluetooth communication as well.

Check out [uwp-socket](https://github.com/MikeKovarik/uwp-socket). Work in progress though, not enough free time.

**Comment by gaspardip** [comment]
Thats fair. I have had lots of problems establishing a tcp connection using uwp js (because of the lack of documentation at the time, and a tested net module like node). I've used this, now deprecated, module in the past and had to include JSPM and yeah it was lots of pain.

After giving up on jspm I created a tcpclient wrapper using this repo and node's source, its far from perfect, I know, but its what im using now https://gist.github.com/gaspardip/007bc55c0cc219acd42b32c91f625544. Check specially the `try/catch`s, statements like `new StreamSocket()` can throw an error.
